swagger/index.js

```diff
@@ -100,25 +100,25 @@ module.exports = yeoman.generators.Base.extend({
       var modelDef = {
         name: swaggerModel,
         http: {
-          path: basePath
+          path: basePath,
         },
         base: 'Model',
         facetName: 'server', // hard-coded for now
-        properties: {}
+        properties: {},
       };
       api.modelDefinition = modelDef;
       self.modelDefs.push(modelDef);
       self.modelConfigs.push({
         name: swaggerModel,
         facetName: 'server', // hard-coded for now
         dataSource: null,
-        public: true
+        public: true,
       });
     }
 
     for (i = 0, n = self.apis.length; i < n; i++) {
       api = self.apis[i];
-      for (var m in api.models) {
+      for (m in api.models) {
```

Comment:
Please keep the definition of `var m` here. There is a bug in eslint where it reports false warning, the workaround is to temporarily disable the rule for this line. Search for "one-var" in https://github.com/strongloop-internal/loopback-knowledge-base/blob/4b0aacb99501e3d6d4b31a52685e1d14ab268c68/eslint-guide.md